src/main/java/org/spongepowered/api/event/cause/EventContextKeys.java

```diff
@@ -106,6 +106,65 @@
 
     // SORTFIELDS:OFF
 
+    public static final class Named {
+        // SORTFIELDS:ON
+
+        public static final String BLOCK_HIT = "BLOCK_HIT";
```

Comment:
Should just make these names be the ids as in https://github.com/SpongePowered/SpongeCommon/blob/bleeding/src/main/java/org/spongepowered/common/registry/type/event/EventContextKeysModule.java#L78